python/sglang/test/runners.py

```diff
@@ -130,7 +130,7 @@ def start_model_process(self, in_queue, out_queue, model_path, torch_dtype):
             self.base_model = AutoModelForCausalLM.from_pretrained(
                 model_path,
                 torch_dtype=torch_dtype,
-                trust_remote_code=False,
+                trust_remote_code=True,
```

Comment:
Is this change necessary? Many other tests use this code. It would be better to keep the original version.

Reply:
(see above)